runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -160,9 +162,10 @@ class ClientSessionData
    // The following hashtable caches <classname> --> <J9Class> mappings
    // All classes in here are loaded by the systemClassLoader so we know they cannot be unloaded
    PersistentUnorderedMap<std::string, TR_OpaqueClassBlock*> _systemClassByNameMap;
- 
+   PersistentUnorderedMap<J9Class *, UDATA *> _classChainDataCacheMap;
```

Comment:
Name could be better here: delete either `Cache` or `Map` because they mean more or less the same thing.